lib/thermos.rb

```diff
@@ -8,14 +8,30 @@
 require 'thermos/rebuild_cache_job'
 
 module Thermos
-  def self.keep_warm(key:, model:, id:, deps: [], lookup_key: nil, filter: nil, &block)
-    fill(key: key, model: model, deps: deps, lookup_key: lookup_key, filter: filter, &block)
+  def self.keep_warm(key:, model:, id:, deps: [], lookup_key: nil, filter: nil, queue: nil, &block)
```

Comment:
Curious if there's a reason why we aren't specifying the default queue in this case? The default in Beverage is a catch-all but still just wondering why we're specifying it for `fill` but not for `keep_warm`.

Reply:
going to remove from 'fill', wanted to just supply in the singular place in the `Beverage` class